<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mensaje</class>
 <widget class="QDialog" name="Mensaje">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mensaje Error!</string>
  </property>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_ok">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titulo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Error!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
